python/gtsam/utils/plot.py

```diff
@@ -75,8 +75,9 @@ def plot_covariance_ellipse_3d(axes,
     Plots a Gaussian as an uncertainty ellipse
 
     Based on Maybeck Vol 1, page 366
-    k=2.296 corresponds to 1 std, 68.26% of all probability
-    k=11.82 corresponds to 3 std, 99.74% of all probability
+    For the 3D case:
+    k = 3.527 corresponds to 1 std, 68.26% of all probability
+    k = 14.157 corresponds to 3 std, 99.74% of all probability
```

Comment:
Do you have an online reference?

Reply:
With code from other function:
pct_to_sigma(sigma_to_pct(1, 1), 3) ** 2 --> 3.5267403802617303
pct_to_sigma(sigma_to_pct(3, 1), 3) ** 2--> 14.156413609126677

So seems about right if we want this behavior.